src/com/company/Main.java

```diff
@@ -1,8 +1,41 @@
 package com.company;
+import java.util.Scanner;
 
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+        Scanner sc = new Scanner(System.in);
+        int carNumber = 0;
+        int hitNumber = 0;
+
+        while(true) {
+            System.out.println("자동차 대수는 몇 대 인가요?");
+            carNumber = sc.nextInt();
+            car[] car1 = new car[carNumber];
+
+            for (int i = 0; i < car1.length; i++) {
+                car1[i] = new car();
+            }
```

Comment:
인덱스가 필요 없을땐 foreach를 사용하면 더 편하게 쓸 수 있음

```suggestion
            for (Car car : car1) {
                car = new car();
            }
```